shaper-core/src/test/kotlin/dev/icerock/tools/shaper/core/ShaperTest.kt

```diff
@@ -25,6 +25,7 @@ class ShaperTest {
         val config = TemplateConfig(
             globalParams = mapOf("packageName" to "dev.icerock"),
             files = listOf(buildGradleFile, sourceCodeFile),
+            includes = listOf("src/test/resources/includes", "src/test/resources/second-includes"),
```

Comment:
why we can't use here `listOf("includes", "second-includes")`?

Reply:
Fix it